ts/features/uaDonations/screens/UAWebViewScreen.tsx

```diff
@@ -81,30 +77,60 @@ const handleOnMessage = (
     JSON.parse(event.nativeEvent.data)
   );
   if (maybeMessage.isLeft()) {
-    handleError("decoding error: " + readableReport(maybeMessage.value));
+    void mixpanelTrack("UADONATIONS_WEBVIEW_DECODE_ERROR", {
+      reason: `decoding error: ${readableReport(maybeMessage.value)}`
+    });
+    handleError();
     return;
   }
   switch (maybeMessage.value.kind) {
     case "webUrl":
       const webUrl = maybeMessage.value.payload;
+      void mixpanelTrack("UADONATIONS_WEBVIEW_OPEN_WEBURL_REQUEST", {
+        url: webUrl
+      });
       openWebUrl(webUrl, () => {
-        handleError("webUrl error: " + webUrl);
+        void mixpanelTrack("UADONATIONS_WEBVIEW_OPEN_WEBURL_ERROR", {
+          url: maybeMessage.value.payload
+        });
+        handleError();
       });
       break;
     case "payment":
       const { nav, cf, amount } = maybeMessage.value.payload;
+      void mixpanelTrack("UADONATIONS_WEBVIEW_PAYMENT_REQUEST", {
+        organizationFiscalCode: cf,
+        paymentNoticeNumber: nav,
+        amount
+      });
       const maybeRptId = RptId.decode({
         paymentNoticeNumber: nav,
         organizationFiscalCode: cf
       });
       const maybeAmount = AmountInEuroCents.decode(amount.toString());
       if (maybeRptId.isLeft() || maybeAmount.isLeft()) {
+        const reason = maybeRptId.isLeft()
+          ? maybeRptId.value
+          : maybeAmount.value;
+        void mixpanelTrack("UADONATIONS_WEBVIEW_PAYMENT_DECODE_ERROR", {
+          reason
+        });
+        handleError();
         return;
       }
+      void mixpanelTrack("UADONATIONS_WEBVIEW_PAYMENT_DATA_SUCCESS", {
+        organizationFiscalCode: maybeRptId.value.organizationFiscalCode,
+        paymentNoticeNumber: nav,
+        amount: maybeAmount.value
+      });
```

Comment:
```suggestion
      void mixpanelTrack("UADONATIONS_WEBVIEW_PAYMENT_DECODE_SUCCESS", {
        organizationFiscalCode: maybeRptId.value.organizationFiscalCode,
        paymentNoticeNumber: PaymentNoticeNumber.encode(
          maybeRptId.value.paymentNoticeNumber
        ),
        amount: maybeAmount.value
      });
```
just a little refactoring to make the event type more explicit and to avoid to have a plain object

before
```
{"amount": "10000", "organizationFiscalCode": "11111111111", "paymentNoticeNumber": {"auxDigit": "3", "checkDigit": "94", "iuv13": "0416467289224", "segregationCode": "00"}}
```

now
```
{"amount": "1000", "organizationFiscalCode": "11111111111", "paymentNoticeNumber": "300011646728922494"}
```